### ISSUE TITLE
Add more bastard swords, unique weaponry, etc. Longswords and certain item types are too good/common

### ISSUE THREAD

**Issue by b5635** [b5635/the-frozen-north#612]
LootaBox — Today at 11:14 AM
Would it make sense to change some of the better martial weapons into exotic ones? E.g. I've encountered several good longswords and greatswords, even some heavy flails and I think a halberd.. but hardly any bastard swords, katanas, dwarven waraxes or scythes. They appear much less frequently and seem to never be quite up to par with the martial cousins
pok — Today at 11:15 AM
Bastard swords and katanas would be good
Less so for scythes scimitars and rapiers because they are extra good for WMs
LootaBox — Today at 11:15 AM
Given that feats are really contested with low lvls, its hard to justify taking exotic weapon proficiency currently, or so it seems to me 
pok — Today at 11:16 AM
I can look into adding a lot more exotic weapons for sure
Bastard swords should be the same type i.e. common
LootaBox — Today at 11:16 AM
I don't think adding more of them will solve it - there's just so good martial weapons available
pok — Today at 11:16 AM
But the other exotic ones are more rare. Or maybe bastard swords are uncommon tier?
LootaBox — Today at 11:17 AM
I mean, more is more 👌 
But still hard to justify taking exotic weapon proficiency when you can get a busted longsword or greatsword more readily
Loggy the Clueless Goose — Today at 11:17 AM
iirc bastard swords are uncommon
pok — Today at 11:17 AM
Yeah I should make bastard swords common then and perhaps shift some of them over
Bastard swords should be equivalent to longswords tbh, so I can at least look into some good ones for that
Loggy the Clueless Goose — Today at 11:18 AM
some of them do get quite good
pok — Today at 11:19 AM
Admittedly there are great longswords. And I think that's okay, because that what they were like originally in the games

Generally speaking, I made things in a way that if you chose a specific item to specialize, you will have an easier time
Loggy the Clueless Goose — Today at 11:20 AM
some of the issue here may be that there is basically no way to reliably see t5 uniques 
pok — Today at 11:20 AM
Instead of nerfing scimitars (which they are already nerfed) they're somewhat less common in general
Yeah I think we would need a high level quest, area, or infinite dungeons
Loggy the Clueless Goose — Today at 11:20 AM
like for weapons where +3 isn't the best thing possible, seeing the t5 uniques that are better than generic +3s isn't really possible 
and the generic +3s are 3x as common as any random unique, which is already really rare
pok — Today at 11:21 AM
Hmm, it's not a 50% split?
Between unique and non unique
Loggy the Clueless Goose — Today at 11:22 AM
const int UNIQUE_ITEM_CHANCE = 33;
pok — Today at 11:22 AM
I wonder if it would be better to change it to 50
Or some higher number

LootaBox — Today at 11:23 AM
Greatsword is the highest damaging weapon on its own already, and it really seems the best magic 2 hander are all greatswords - Whenever I'm thinking I want to make a two-hander I end up thinking it has to be greatsword or I'm gimping myself

For 1h its more contested, but I'd say longsword, rapier and daggers are up there...

Other weapon types have good stuff, but either are not as good by base stats, are more rare or require a feat
E.g., lets say there's a really neat unique 1d8 two-hander that has like +2AB, +1d6 dmg and maybe a chance stun or whatever... Compare that to your regular generic 2d6 Greatsword +2

The greatsword will deal more damage, is probably cheaper and usable at a lower level. Likely not so rare either, currently
LootaBox — Today at 11:32 AM

I do think adding more items to the other weapons would be good, because it will increase their pool and if a non unique item is selected, there's a higher chance to get these items
And they may need some good ones anyways
I could also add in some crappier longswords too 😉
Which theoretically decreases the chance of good longswords. Or good "common" weapons in general

**Comment by b5635** [comment]
TL;DR

There are very good longswords and/or certain item types, enough that it isn't worthwhile to get other, more rare items such as bastard swords, etc.

The fix will probably be to add in some more items of those types so it 1) increases the pool and 2) gets some very decent alternatives as well.

We may need to investigate changing the ratio of unique items as well
